src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -96,6 +96,10 @@ const ReportActionItemFragment = (props) => {
             }
             const {html, text} = props.fragment;
 
+            if (props.fragment.isDeletedParentAction) {
+                return <RenderHTML html={`<comment>${props.translate('parentReportAction.deletedMessage')}</comment>`} />;
+            }
```

Comment:
NAB: Will this show "deleted message" for each report action that is under a deleted thread?

Why we are showing report actions in this case? Wouldn't it be better to just show one message that the thread is deleted (and not render the report actions) Sorry if this was discussed already, just asking.

Reply:
For each report action, if the reportAction has a child then we will render [Message Deleted]. Therefore, you can still open the thread replies if the original parent message is deleted

Reply:
Oh I got it so this more like `hasChild & isDeleted`, the variable name is a bit confusing. Thanks for the clarification.